code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -87,17 +91,72 @@
 		beaker = null
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/update_icon()
-	if(panel_open)
-		icon_state = "cell-o"
-	else if(state_open)
-		icon_state = "cell-open"
-	else if(on && is_operational())
-		if(occupant)
-			icon_state = "cell-occupied"
+	cut_overlays()
+
+	if(state_open)
+		icon_state = "pod-open"
+	else if(occupant)
+		var/mutable_appearance/occupant_overlay

```

Comment:
I had the impression from #26518 that mutable appearances used as overlays, like these are, can't have a different dir from the pod itself.